test/wasm2c/wast/stack_pointer.wast

```diff
@@ -0,0 +1,61 @@
+(module
+  (type (;0;) (func (param i32 i32)))
+  (type (;1;) (func (param i32 i32) (result i32)))
+  (func $store_char*__char_ (type 0) (param i32 i32)
+    local.get 0
+    local.get 1
```

Comment:
The extra/redundant local.get/local.set instructions look like they might be coming from unoptimized llvm output?   Perhaps that test would be more readable if use use `-O1` or `-O2` or run `wasm-opt` on the binary?   

Ideally we could hand-write a smaller/simpler/more readable test?